src/listen/heosResponse.ts

```diff
@@ -0,0 +1,40 @@
+import { HeosResponse, HeosResponseMessage } from '../types'
+import {
+	FIRST_LEVEL_HEOS_RESPONSE_MESSAGE_SPLITTER,
+	SECOND_LEVEL_HEOS_RESPONSE_MESSAGE_SPLITTER
+} from '../utils/constants'
+
+export function parseHeosMessageString(
+	messageString: string | undefined
+): HeosResponseMessage | string | void {
+	if (typeof messageString === 'string') {
+		const messageObject: HeosResponseMessage = {}
+		let keyAndValuePairs: string[] = messageString.split(
+			FIRST_LEVEL_HEOS_RESPONSE_MESSAGE_SPLITTER
+		)
+
+		// A single message (without message1&message2 pattern)
```

Comment:
What does this do? I do not fully understand it

Reply:
It does nothing, redundant lines